### ISSUE TITLE
Value Sets - UI updates

### ISSUE THREAD

**Issue by a-chappelear** [CDCgov/prime-reportstream#6959]
## Problem Background
Based on usability feedback on value sets, users would like to see a method for confirming a change and a warning that the change will alter settings.

**Other UI items to consider**
- moving "add new" to the top of the page
- clarifying in content what the change will alter (or how, it isn't clear that this change could be overwritten in code as well, which overwrites the UI)

## Reference Materials
- Current location of value sets: https://reportstream.cdc.gov/admin/value-sets
- Feedback from O+O can be found in #6787 

**AC:**
- [ ] Updated wireframes that are reviewed with the experience and O&O teams prior to to any development work
- [ ] When adding new row, move those entry fields to the top of the table instead of the bottom
- [ ] Change confirmation dialogue box
- [ ] Warning that the change will alter settings

**Comment by drew-usds** [comment]
@HeatherRienks From design sync 11/24: worth keeping this in the backlog. Could become part of a larger epic to better align how the admin section/tools work.

**Comment by kevinhaube** [comment]
@drew-usds Can we narrow the scope for a moment and think about whether there's easy wins here for Value Sets? Lots of good feedback in #6787 on ways we can improve not just the UI but the service as a whole. Metadata on table changes, managing race conditions between two updates to the same table, etc.

**Comment by chris-kuryak** [comment]
@brick-green @GilmoreA6 @oslynn @MauriceReeves-usds 

- Is this still a problem? 
- A big pain point? 
- Do y'all still need this fix to the Value Sets tool?

cc: @brandonnava

**Comment by chris-kuryak** [comment]
blocked pending above response.

**Comment by MauriceReeves-usds** [comment]
@chris-kuryak it's probably lower priority right now, but still important, especially as we add new conditions and want to add the ability to manage data like this in the UI. It was a higher priority when we were managing the CSV uploader directly.

**Comment by chris-kuryak** [comment]
I have unblocked due to answer from Mo above. For awareness, I converted this to an epic with two issues, one for the design work and one for the engineering work. Since it is low priority, I have added that label. We will begin refining these after review with the Experience team.
